awx/ui/client/src/templates/job_templates/multi-credential/multi-credential-modal.directive.js

```diff
@@ -111,7 +111,7 @@ function multiCredentialModalController(GetBasePath, qs, MultiCredentialService)
 
         scope.credentialTypes.forEach((credentialType => {
             if(credentialType.kind
-                .match(/^(machine|cloud|net|ssh|vault)$/)) {
+                .match(/^(machine|cloud|net|ssh|vault|kubernetes)$/)) {
```

Comment:
We'll need to make sure the new option gets displayed on the multicredential select lookup in ui_next:
![cred_form](https://user-images.githubusercontent.com/9753817/87679688-7a34a800-c74a-11ea-8c49-a876515015dc.gif)

... which I presume to be something to be added [here](https://github.com/ansible/awx/blob/devel/awx/ui_next/src/components/Lookup/CredentialLookup.jsx#L81)